packages/clay-button/src/ClayButton.soy

```diff
@@ -164,7 +164,7 @@
 	{/if}
 
 	{if $disabled}
-		disabled
+		disabled="true"
```

Comment:
According to disabled docs this should be `disabled="disabled"`.

Source: https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/Attribute/disabled

Thx!

Reply:
This makes more sense now. Thanks @carloslancha.